### PR TITLE
shikhareddy

### PR DESCRIPTION
**Pull request by shikhareddy**
Hello sir
Did the basic changes in README.md file. Please let me know if any changes are to be made.
Moving on to the docstrings.

### CODE REVIEW

**Comment by chiragnagpal** [comment]
Hi @shikhareddy don't replace the inline latex with unicode characters, instead try to see if inline latex within markdown renders correctly on github. Check this blog post for hints:
http://flennerhag.com/2017-01-14-latex/

If that works, update this pull request.

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Deep Convolutional Survival Machines
 ------------------------------------
 
 Predictive maintenance and medical imaging sometimes requires to work with
-image streams. Deep Convolutional Survival Machines extends **DSM** and
+image streams. Deep Convolutional Survival Machines extend **DSM** and
```
Comment (chiragnagpal) [inline README.md:59]: DSM, DCSM and DRSM are all singular quantities

File: README.md [hunk 2]
```diff
@@ -93,16 +93,16 @@ To evaluate performance using standard metrics
 
 Contributing
 ------------
-`dsm` is [on GitHub]. Bug reports and pull requests are welcome.
+`dsm` is on [GitHub]. Bug reports and pull requests are welcome.
```
Comment (chiragnagpal) [inline README.md:96]: Dont change this

**Comment by chiragnagpal** [comment]
@shikhareddy  I just rechecked, I think github doesnt support latex in markdown,  😱 , so ignore my comment about that

**Comment by shikhareddy** [comment]
@chiragnagpal okay so do i keep the unicode characters or change it to the way it was originally?

**Pull request closed without merging**